src/worker.ts

```diff
@@ -208,7 +208,9 @@ if (isNodeProd()) {
 		},
 		lifetime: Infinity
 	});
-} else {
+
+	//Don't start worker automatically in test environments to allow calling start() and stop() functions in unit tests
+} else if (!isNodeTest()) {
```

Comment:
So, this change needs some advice from @mboudreau

I want to have an end-to-end integration test which will invovle the queue. For that purpose I had to use "Start" and "stop" functions from this file to start worker in unit tests.

Unfortunately we can't just import them, because as soon as we do it this code gets executed. So I've added a check to make sure it won't start automatically in the test environment, so we can use it in unit tests.

Another way of doing it is to extract queues startup code (start() and stop() functions to a separate file, but I wanted to avoid changing too much code in this PR. So I am doing this way.

I wonder if it can affect something? (Do we use test env when we run it in Docker?)

Reply:
Had some chat with @tarakpurohit, we decided to move start() and stop() function to a seaparate file

Reply:
- [ ] extract start and stop funcitons from the worker entry point code